### PR TITLE
Vancelot/inspec migration

### PR DESCRIPTION
**Pull request by Vancelot11**
Converge and verifies successfully.

### CODE REVIEW

**Comment by Vancelot11** [comment]
[Migrate from ServerSpec to InSpec](https://trello.com/c/2EA1WbBL/1082-migrate-from-serverspec-to-inspec)

File: test/integration/default/inspec/default_test.rb [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  its(:exit_status) { should eq 0 }
+end
+
+%w(33000 33001 34000 34001).each do |p|
+  describe port(p) do
+    it { should be_listening }
+  end
+end
+
+describe command('docker ps --format "{{.Names}}:{{.Status}}"') do
```
Comment (ramereth) [inline test/integration/default/inspec/default_test.rb]: Please change this to the ``docker_container`` resource.

File: test/integration/default/inspec/default_test.rb [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+end
+
+%w( start-container.sh start-mysql.sh ).each do |file|
+  script = ::File.join(script_path, file)
+  describe command("#{script} #{pass_csv_path} 00") do
+    its(:exit_status) { should eq 0 }
+  end
+end
+
+describe command('docker ps --format "{{.Names}}:{{.Status}}"') do
```
Comment (ramereth) [inline test/integration/default/inspec/default_test.rb]: Please change this to the ``docker_container`` resource.

File: test/integration/default/inspec/default_test.rb [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  its(:stdout) { should match(/^dobc-0[01]:Up/) }
+  its(:stdout) { should match(/^mysql-0[01]:Up/) }
+end
+
+script = ::File.join(script_path, 'stop-all.sh')
+describe command("#{script} #{pass_csv_path}") do
+  its(:exit_status) { should eq 0 }
+end
+
+describe command('docker ps --format "{{.Names}}:{{.Status}}"') do
```
Comment (ramereth) [inline test/integration/default/inspec/default_test.rb]: Please change this to the ``docker_container`` resource.

**Review by ramereth — CHANGES_REQUESTED** [review]
You also need to remove the old ServerSpec tests.

**Review by ramereth — APPROVED** [review]
Great work! Since you changed one of the files, please do a patch bump by doing ``!bump patch ~``.

**Comment by Vancelot11** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v2.0.1.  Have a nice day!

**Pull request merged**